src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -194,6 +190,31 @@ def _validate_target_field(
         default=None,
     )
 
+    mixins: Optional[Dict[str, Dict]] = Field(
+        title="Mixins to apply to entities",
+        default=None,
+    )
+
+    @model_validator(mode="before")
+    @classmethod
+    def apply_mixins(cls, data: Dict) -> Dict:
+        """
+        Applies mixins to those entities, whose meta field contains the mixin name.
+        """
+        if "mixins" in data and "entities" in data:
```

Comment:
```suggestion
        if "mixins" not in data or "entities" not in data:
           return data
```

Less indentation later

Reply:
Done